airflow/operators/dummy_operator.py

```diff
@@ -24,6 +24,9 @@ class DummyOperator(BaseOperator):
     """
     Operator that does literally nothing. It can be used to group tasks in a
     DAG.
+
+    The task that uses this operator is automatically marked as successfully completed by the scheduler
```

Comment:
@mik-laj This is what confused me.
Maybe better saying something like:
The task is evaluated by the scheduler but never processed by the executor
WDYT?

Reply:
Yeah. Might be worth adding!

Reply:
I updated the description.